fflogsapi/util/filters.py

```diff
@@ -1,5 +1,21 @@
 from typing import Any
 
+def construct_filter_expression_string(expressions: dict[str,Any]) -> str:
+    '''
+    Construct a filter expression from a dicttionary of expressions
+    that can be used in FFLog queries to filter values base on FFLogs Expressions.
+    
+    Args:
+        expressions: A dictions of keys which are fields to filter,
+                     and values are the values to filter by.
+    Returns:
+        An expression string usable in FFLogs expressions.
+    '''
+    prepped_expressions = []
+    for key, e in expressions.items():
+        expression = f'{key}=\'{e}\''
```

Comment:
This should support additional comparisons (<, >, <=, >=, =, !=), or add a warning to the docstring that only strict equality on strings is supported at the moment